prow/create-deps-cm.sh

```diff
@@ -77,7 +79,7 @@ check_getopt() {
 get_options() {
   check_getopt
 
-  if opt=$(getopt -o b:c:n:k:i -l branch:,context:,namespace:,key:,interactive -n "$script_name" -- "$@"); then
+  if opt=$(getopt -o b:c:n:k:l:i -l branch:,context:,namespace:,key:,local,interactive,dry-run -n "$script_name" -- "$@"); then
```

Comment:
This would be a lot easier to read (and therefore maintain) if it is written in python or golang

Reply:
I will rewrite in python.